README.md

```diff
@@ -1 +1 @@
-
+[https://GITHUB-GevinduH.github.io/ cv”/cv](cv.md)
```

Comment:
Is there a typo in the link? Or is it correct?

Reply:
I have fixed the description of the task in our docs. The correct link to your exercise should be: https://gevinduh.github.io/CV-Repo/cv

Reply:
I have updated the exercise description in our docs. The correct link for you is: https://gevinduh.github.io/CV-Repo/cv

Reply:
@vieraboschkova Fixed as discussed